test/e2e/specs/linksConsoleErrorsEsmPage.spec.ts

```diff
@@ -1,19 +1,26 @@
-import { ConsoleMessage, expect } from '@playwright/test';
+import { ConsoleMessage, expect, Page } from '@playwright/test';
 import { vpTest } from '../fixtures/vpTest';
 import { ESM_LINKS } from '../testData/esmPageLinksData';
 import { waitForPageToLoadWithTimeout } from '../src/helpers/waitForPageToLoadWithTimeout';
 import { validatePageErrors } from '../src/helpers/validatePageErrors';
+import { ExampleLinkType } from '../types/exampleLinkType';
+
+const EDGE_ESM_URL = 'https://cld-vp-esm-pages.netlify.app/';
+// On PR level it will use the preview deploy URL and locally it will use the latest EDGE.
+const ESM_URL = process.env.PREVIEW_URL || EDGE_ESM_URL;
+// Flag to indicate if the deploy preview URL is ready
+let PREVIEW_URL_LOADED = false;
```

Comment:
since this is a changing variable i think it is better to name it like `isPreviewURLLoaded`... since a snake case is often associated with constant variables

Reply:
Changed. Thanks